content/blog/2019/istio1.1_perf/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cloud service management comes of age with Istio 1.1 performance update
```

Comment:
Any way you could shorten the title a bit? This is awful long in the left-hand side bar.

Also, please capitalize titles